code/frameworks/nextjs/src/config/webpack.ts

```diff
@@ -42,6 +42,9 @@ const setupRuntimeConfig = (baseConfig: WebpackConfig, nextConfig: NextConfig):
    * In Next13, the `newNextLinkBehavior` option now defaults to truthy (still
    * `undefined` in the config), and `next/link` was engineered to opt *out*
    * of it
+   *
+   * TODO: Revisit logic as soon as the changes in https://github.com/vercel/next.js/pull/42623
```

Comment:
They got released in `v13.0.6`

Reply:
@yannbf Thanks! [Updated](https://github.com/storybookjs/storybook/pull/20065/commits/db814f270fbad246ae471de602ea40978c264a1b)

Reply:
@yannbf Thanks! Code [adjusted](https://github.com/storybookjs/storybook/pull/20065/commits/6613e778cc4452ce1626fbeac799ca1f5d7e89a2)